connector_prestashop/models/partner.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields, api
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    prestashop_bind_ids = fields.One2many(
+        comodel_name='prestashop.res.partner',
+        inverse_name='openerp_id',
```

Comment:
@guewen What do you think about renaming now to `odoo_id`? Will it interfere with your refactoring?